test/standalone-constraint-constraint/invalid.annotation-invalidRanges.ttl

```diff
@@ -0,0 +1,81 @@
+PREFIX ex: <http://example.org/>
+PREFIX dcterms: <http://purl.org/dc/terms/>
+@prefix relation: <https://cube.link/relation/> .
+@prefix meta: <https://cube.link/meta/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix cube: <https://cube.link/> .
+@prefix observation: <https://environment.ld.admin.ch/foen/nfi/observation/max_min_undefined> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix schema: <http://schema.org/> .
+@base <https://example.org/> .
+
+<cube> a cube:Cube ;
+    cube:observationConstraint <shape> ;
+    cube:observationSet <observationSet> .
+
+<observationSet> cube:observation <observationA> .
+
+<observationA> a cube:Observation ;
+    cube:observedBy <observer> ;
+    <measure> 4.9 ;
+    <year> <https://ld.admin.ch/time/year/2020> ;
+.
+
+<shape> a cube:Constraint ;
+    sh:targetClass cube:Observation ;
+    sh:closed true ;
+    sh:property
+        [
+            sh:path rdf:type ;
+            sh:nodeKind sh:IRI ;
+            sh:minCount 1 ;
+            sh:maxCount 1
+        ] ;
+    sh:property
+        [
+            sh:path cube:observedBy ; ;
+            sh:nodeKind sh:IRI ;
+            sh:minCount 1 ;
+            sh:maxCount 1
+        ] ;
+    sh:property
+        [
+            a cube:MeasureDimension ;
+            sh:datatype xsd:decimal ;
+            sh:path <measure> ;
+            schema:name "measure" ;
+            sh:minCount 1 ;
+            sh:maxCount 1 ;
+            meta:annotation
+                [
+                    a meta:Limit ;
+                    schema:value 97 ;
+                    schema:name "Target 2020+" ;
+                    meta:annotationContext
+                        [
+                            sh:path <year> ;
+                            sh:minInclusive <https://ld.admin.ch/time/year/2020> ;
```

Comment:
why two lower bounds?

Reply:
That's a test cases which should fail

Reply:
I tried to make this even more accurate but I seem to have hit a limitation of shat SHACL can do when there are multiple properties with same value. I failed to make a validation to catch for example 

```turtle
 sh:minInclusive <https://ld.admin.ch/time/year/2020>
 sh:minExclusive <https://ld.admin.ch/time/year/2020>
```